### ISSUE TITLE
Does it able to learn certain voice style?

### ISSUE THREAD

**Issue by lucasjinreal** [ga642381/FastSpeech2#5]
Does it able to learn certain voice style?

**Comment by ga642381** [comment]
Hi, thanks for your question. This repo doesn't support learning voice style for now. We might need a style encoder if we want to learn the voice style. Recently, instead, we have been focusing on multilingual TTS. such as supporting Chinese, Taiwanese, and so on.

**Comment by lucasjinreal** [comment]
@ga642381 hi, does multilane tts performant can compatible with single lan? isn't the phone space would be very large?

**Comment by ga642381** [comment]
I agree with you. So the collaborator of this repo, Wei-Ping Huang, does have some research on how to use self-supervised features to learn shared phonetic information across different languages. (ref: Few-Shot Cross-Lingual TTS Using Transferable Phoneme Embedding https://arxiv.org/abs/2206.15427) 

As for this repo, I think at least we can support different datasets for various languages to make it more friendly for the community to do multispeaker, multilingual TTS research.
